tools/schematics/dt-component-e2e/index.ts

```diff
@@ -23,7 +23,7 @@ import {
   template,
   Tree,
   url,
-} from '@angular-devkit/schematics';
+} from '@nrwl/workspace/node_modules/@angular-devkit/schematics';
```

Comment:
wrong import path never import from node_modules revert this change

Reply:
I removed all sub-node_module imports.